src/dmd/cparse.d

```diff
@@ -3382,8 +3382,21 @@ final class CParser(AST) : Parser!AST
                     break;
 
                 case TOK.leftParenthesis:
-                    if (!skipParens(t, &t))
+                    if (!skipParens(t, &t)) {
```

Comment:
1. Please match the brace style of the rest of the code.
2. Please avoid using ! if practical. Rewrite `if (!C) A; else B;` as `if (C) B; else A;`